packages/frontend/src/components/common/PrivateRoute.js

```diff
@@ -3,25 +3,36 @@ import { connect } from 'react-redux';
 import { Route, withRouter, Redirect } from 'react-router-dom';
 
 import { KEY_ACTIVE_ACCOUNT_ID } from '../../utils/wallet';
-import { GuestLanding } from '../landing/GuestLanding';
+import NoIndexMetaTag from './NoIndexMetaTag';
 
-const PrivateRoute = ({component: Component, account, refreshAccountOwnerEnded, ...rest}) => (
-    <Route 
-        {...rest} 
-        render={(props) => (
-            !localStorage.getItem(KEY_ACTIVE_ACCOUNT_ID)
-                ? (
-                    <Redirect
-                        to={{
-                            pathname: '/',
-                        }}
-                    />
-                )
-                : !account.accountId
-                    ? refreshAccountOwnerEnded && <GuestLanding />
-                    : <Component {...props} />
-        )}
-    />
+const PrivateRoute = ({
+    component: Component,
+    render,
+    account,
+    refreshAccountOwnerEnded,
+    indexBySearchEngines,
+    ...rest
+}) => (
+    <>
+        {!indexBySearchEngines && <NoIndexMetaTag />}
+        <Route 
+            {...rest} 
+            render={(props) => (
+                !localStorage.getItem(KEY_ACTIVE_ACCOUNT_ID)
+                    ? (
+                        <Redirect
+                            to={{
+                                pathname: '/',
+                            }}
+                        />
+                    )
+                    : ( Component // <Route component> takes precedence over <Route render></Route>
+                        ? <Component {...props} />
+                        : (render ? render(props) : <></>)
+                    )
+            )}
```

Comment:
This ternary tree is a little bit out of hand; would be nice to go ahead and replace all of these ternaries with some more readable / less dense code

Webstorm intentions unwrapped the ternaries for me automatically (indentation is likely wrong on this suggestion)
```suggestion
            render={(props) => {
                    if (!localStorage.getItem(KEY_ACTIVE_ACCOUNT_ID)) {
                        return <Redirect
                            to={{
                                pathname: '/',
                            }}
                        />;
                    }
                    
                    // <Route component> takes precedence over <Route render></Route>
                    if (Component) {
                        return <Component {...props} />;
                    }

                    if (render) {
                        return render(props);
                    } 

                    return (<></>);
}}
```